docs/swagger2.yml

```diff
@@ -448,15 +396,16 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /apps/{app}/calls:
+  /apps/{app}/fn/{fn}/calls:
```

Comment:
not sure we should nest this deep for calls. if we leave them on app you can list calls on an app, which may come from various triggers (and thus functions), for causality this could be useful e.g. function A calls function B. we can further sort down by trigger or function or image, but leaving these hanging off of an app allows the additional flexibility there.

Reply:
The idea behind pushing these down is to account for the scale of calls that will occur at an app level. Why deployed this data will most likely be shipped to a dedicated metrics store, which will be interacted with directly.

Putting it down will give cli users something to work with and something that can be supported at scale.